apps/docs/src/local-ts/distribution.md

```
@@ -0,0 +1,103 @@
+# Distribution
+
+Once your app is ready, you will package and distribute it so users can install and run it. 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)